templates/webapi/layouts/navbar.html.ep

```diff
@@ -97,7 +97,9 @@
                   % }
                   %= link_to "Appearance" => url_for('appearance') => class => 'dropdown-item'
                   %= link_to "API help" => url_for('api_help') => class => 'dropdown-item'
-                  %= link_to 'Changelog' => url_for('changelog') => class => 'dropdown-item'
+                  % if ($current_version) {
```

Comment:
Taking https://github.com/os-autoinst/openQA/pull/6396/files#r2048260791 in mind, I would say that this element should be always displayed.

Reply:
Only from rpm packaging. The .spec is where version.txt is created.

I wonder if the version.txt could be prepared as part of the build system, though?

Reply:
I created it in the .spec file because there we have the version available as rpm macro/variable. We could of course pass it from the .spec file to the Makefile via an env variable. I find it a bit of an overcomplication, though.